cou/steps/__init__.py

```diff
@@ -251,6 +251,15 @@ def add_step(self, step: BaseStep) -> None:
 
         self._sub_steps.append(step)
 
+    def add_steps(self, steps: Iterable[BaseStep]) -> None:
```

Comment:
Using `plan.sub_steps.extend` is working fine, but that will not skip an empty step in list of steps.